lib/rules/property-no-unknown/__tests__/index.js

```diff
@@ -360,3 +360,54 @@ testRule({
 		},
 	],
 });
+
+testRule({
+	ruleName,
+	config: [true, { ignoreAtRules: ['supports', /^my-/] }],
+
+	accept: [
+		{
+			code: '@supports(display: grid) { my-property: 1; }',
+		},
+		{
+			code: '@my-at-rule { foo: 1; }',
+		},
+		{
+			code: '@supports(display:grid) { @media (min-width: 10px) { foo: 1; } }',
+		},
+		{
+			code:
+				'@supports(display:grid) { @media (min-width: 10px) {  @my-property-value: { foo: 1; } } }',
+		},
+		{
+			code: '@my-other-at-rule { a: { foo: 1; } }',
+		},
+	],
+
+	reject: [
+		{
+			code: '@import { a { foo: 1; } }',
```

Comment:
```suggestion
			code: '@media screen { a { foo: 1; } }',
```

Make it more standard.